demo/services/mockWorkspaceFlowRunsApi.ts

```diff
@@ -1,5 +1,5 @@
-import { MockApi } from './MockApi'
-import { FlowRun, GraphNode, RunHistory, StateUpdate } from '@/models'
+import { MockApi } from '@/../demo/services/MockApi'
```

Comment:
@stackoverfloweth side effect of the new linting rule?

Reply:
probably. these demos are a bit strange that they have to go up from `@` (src) to get to the demo folder. It's unfortunate but only in demo code and the benefit to production code outweighs IMO